xbmc/cores/AudioEngine/DSPAddons/ActiveAEDSP.cpp

```diff
@@ -513,7 +513,7 @@ CAEChannelInfo CActiveAEDSP::GetInternalChannelLayout(AEStdChLayout stdLayout)
   return CAEUtil::GetAEChannelLayout(channelLayoutOut);
 }
 
-bool CActiveAEDSP::CreateDSPs(unsigned int &streamId, CActiveAEDSPProcessPtr &process, AEAudioFormat inputFormat, AEAudioFormat outputFormat, bool upmix,
+bool CActiveAEDSP::CreateDSPs(unsigned int &streamId, CActiveAEDSPProcessPtr &process, AEAudioFormat &inputFormat, AEAudioFormat &outputFormat, bool upmix,
```

Comment:
Thats a good idea. 
With that we make sure that the values didn't change by this method.